Essentials/src/main/java/com/earth2me/essentials/commands/Commandrecipe.java

```diff
@@ -111,13 +111,14 @@ public void shapedRecipe(final CommandSource sender, final ShapedRecipe recipe,
             final InventoryView view = user.getBase().openWorkbench(null, true);
             final String[] recipeShape = recipe.getShape();
             final Map<Character, ItemStack> ingredientMap = recipe.getIngredientMap();
+            final boolean legacy = VersionUtil.getServerBukkitVersion().isLowerThan(VersionUtil.v1_13_0_R01);
```

Comment:
Same issue as in Condense, use a global variable like you did in MaterialUtil

Reply:
Same